source/deploying-with-kubernetes/kubernetes-conf.rst

```diff
@@ -0,0 +1,320 @@
+        wazuh-nginx           LoadBalancer   xxx.yy.zzz.223   internal-a3b1...   80:31831/TCP,443:30974/TCP       11m
+        wazuh-workers         LoadBalancer   xxx.yy.zzz.26    internal-a7f9...   1514:31593/TCP                   9m
+
+**Deployments**
+
+    .. code-block:: bash
+
+        $ kubectl get deployments -n wazuh
+        NAME             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+        wazuh-kibana     1         1         1            1           11m
+        wazuh-logstash   1         1         1            1           10m
+        wazuh-nginx      1         1         1            1           11m
+
+**Statefulsets**
+
+    .. code-block:: bash
+
+        $ kubectl get statefulsets -n wazuh
+        NAME                     DESIRED   CURRENT   AGE
+        wazuh-elasticsearch      1         1         13m
+        wazuh-manager-master     1         1         9m
+        wazuh-manager-worker-0   1         1         9m
+        wazuh-manager-worker-1   1         1         9m
+
+**Pods**
+
+    .. code-block:: bash
+
+        $ kubectl get pods -n wazuh
+        NAME                              READY     STATUS    RESTARTS   AGE
+        wazuh-elasticsearch-0             1/1       Running   0          15m
+        wazuh-kibana-f4d9c7944-httsd      1/1       Running   0          14m
+        wazuh-logstash-777b7cd47b-7cxfq   1/1       Running   0          13m
+        wazuh-manager-master-0            1/1       Running   0          12m
+        wazuh-manager-worker-0-0          1/1       Running   0          11m
+        wazuh-manager-worker-1-0          1/1       Running   0          11m
+        wazuh-nginx-748fb8494f-xwwhw      1/1       Running   0          14m
+
+**Accesing Kibana**
+
+    In case you created domain names for the services, you should be able to access Kibana using the proposed domain name: https://wazuh.your-domain.com.
+
+    Also, you can access using the External-IP (from the VPC): https://internal-xxx-yyy.us-east-1.elb.amazonaws.com:443
+
+    .. code-block:: bash
+
+        $ kubectl get services -o wide -n wazuh
+        NAME                  TYPE           CLUSTER-IP       EXTERNAL-IP                                                    PORT(S)                          AGE       SELECTOR
+        wazuh-nginx           LoadBalancer   xxx.xx.xxx.xxx   internal-xxx-yyy.us-east-1.elb.amazonaws.com                   80:31831/TCP,443:30974/TCP       15m       app=wazuh-nginx
+
+.. note::
+    `AWS route 53 <https://aws.amazon.com/route53/?nc1=h_ls>`_ can be used to create a DNS that points to the load balancer and make it accessible through that DNS.
+
+Agents
+------
+
+Monitoring hosts
+^^^^^^^^^^^^^^^^
+
+Wazuh agents are designed to monitor hosts. To start using them:
+
+1. :doc:`Install the agent. <../installation-guide/installing-wazuh-agent/index>`
+
+2. Now, register the agent using the :doc:`registration service <../user-manual/registering/use-registration-service>`, 
+
```

Comment:
link typos?

Reply:
The links are not broken, but they don't work until the docu is made.
In the "GitHub" code viewer, they show like this.

Reply:
Perfect, thanks